darkreader.css

```diff
@@ -27643,7 +27644,9 @@ div.p-workspace__input.p-message_input > div {
 }
 /* Windows Unreads */
 div.c-scrollbar__child > div.c-virtual_list__scroll_container,
-.p-unreads_view__header {
+.p-unreads_view__header,
+.c-virtual_list,
+.p-unreads_view__empty {
```

Comment:
Adding these two classes (`.c-virtual_list` and `.p-unreads_view__empty`) here fixed unread views. 

If the list of unread messages didn't take up the whole screen, the background only applied to the area they filled. Using `.c-virtual_list` just covers the whole thing.

Additionally, if there were no unreads at all, it uses a different element to display a message an image. Using `.p-unreads_view__empty` covers that case as well.